parlai/torchscript/export_model.py

```diff
@@ -0,0 +1,731 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Dict, List, Tuple
+
+import torch.jit
+import torch.nn as nn
+
+from parlai.core.agents import create_agent
+from parlai.core.dict import DictionaryAgent
+from parlai.core.opt import Opt
+from parlai.core.params import ParlaiParser
+from parlai.core.torch_agent import TorchAgent
+from parlai.utils.bpe import Gpt2BpeHelper
+from parlai.utils.io import PathManager
+
+
+def export_model(opt: Opt):
+
+    opt['no_cuda'] = True
+    # TorchScripting is CPU only
+    opt['model_parallel'] = False
+    # model_parallel is not currently supported when TorchScripting
+
+    agent = create_agent(opt, requireModelExists=True)
+
+    inputs = opt['input'].split('|')
```

Comment:
????

Reply:
This is just to provide sample input to test with when scripting